server/routes/api/vote.js

```diff
@@ -0,0 +1,34 @@
+import { Router } from 'express';
+import AricleValidator from '../../middlewares/validations/articleValidator';
+import VoteController from '../../controllers/votes/VoteController';
+import JWTAuthentication  from '../../middlewares/JWTAuthentication';
+import ArticleVoteValidator
+from '../../middlewares/validations/articleVoteValidator';
+
+const voteRouter = Router();
+
+voteRouter.post(
+  '/votes/:articleId/like',
+  JWTAuthentication.authenticateUser,
+  AricleValidator.articleExistInDatabase,
+  ArticleVoteValidator.validateRequest,
+  VoteController.reactToAnArticle,
+);
+
+voteRouter.put(
+  '/votes/:articleId/unlike',
+  JWTAuthentication.authenticateUser,
+  AricleValidator.articleExistInDatabase,
+  ArticleVoteValidator.validateRequest,
+  VoteController.unlikeAnAlreadyLikedOrDislikedArticle,
```

Comment:
can we change this name "unlikeAnAlreadyLikedOrDislikedArticle"?

Reply:
Do you have any name in mind?